src/main/java/org/elasticsearch/index/query/RangeQueryBuilder.java

```diff
@@ -53,6 +66,12 @@
     public RangeQueryBuilder(String name) {
         this.name = name;
     }
+    
+
+    @Inject
+    public RangeQueryBuilder() {
+        this.name = null;
```

Comment:
Open question: There are several places where I had to move no-arg constructor from FooParser to Builder where I had to instantiate a 'final' member to null. At other places I chose to remove the 'final' from the member. Not sure whats better here, maybe this changes in further refactoring anyway?